gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ version-name = "1.4.1"
 accompanistPermissions = "0.33.2-alpha"
 activity = "1.8.0"
 androidxHilt = "1.1.0"
-compose = "1.6.0-alpha08"
+compose = "1.6.0-beta03"
```

Comment:
Great that you cared about updating from the alphas! :) However, I'd recommend to keep updating dependencies to a separate PR so they're easily traceable in case any changes from the latest dependencies breaks the app.

Reply:
Makes sense, should I revert this?